### PR TITLE
TASK: Translate loading status for selectbox component

### PR DESCRIPTION
**Pull request by samuelhauser**
Fix issue #1252

### CODE REVIEW

File: packages/react-ui-components/src/SelectBox/selectBox.js [hunk 1]
```diff
@@ -233,7 +241,7 @@ export default class SelectBox extends PureComponent {
             label = selectedValue.label;
             icon = selectedValue.icon ? selectedValue.icon : icon;
         } else if (displayLoadingIndicator) {
-            label = '[Loading]'; // TODO: localize
+            label = i18nRegistry.translate('loading', 'Loading', [], 'Neos.Neos', 'Main');
```
Comment (dimaip) [inline packages/react-ui-components/src/SelectBox/selectBox.js]: I think it would be simpler to just use a <I18n> component here, no?

**Comment by markusguenther** [comment]
Guess @dimaip meant something like:

```
import I18n from '@neos-project/neos-ui-i18n';
<I18n id="Neos.Neos.Ui:Main:___Loading_label____"/>
```

And then add a new label to the xliff file.

File: packages/neos-ui-editors/src/Editors/Link/index.js [hunk 2]
```diff
@@ -141,6 +141,7 @@ class LinkEditor extends PureComponent {
                 value={this.props.value}
                 onValueChange={this.handleValueChange}
                 placeholder={this.props.i18nRegistry.translate(this.props.options.placeholder)}
+                loadingLabel={this.props.i18nRegistry.translate('loading', 'Loading OLD', [], 'Neos.Neos', 'Main')}
```
Comment (dimaip) [inline packages/neos-ui-editors/src/Editors/Link/index.js]: Loading OLD?

Comment (samuelhauser) [reply packages/neos-ui-editors/src/Editors/Link/index.js]: @dimaip Uups... This was for testing.

File: packages/neos-ui-editors/src/Editors/SelectBox/DataSourceBasedSelectBoxEditor.js [hunk 3]
```diff
@@ -97,12 +97,15 @@ export default class DataSourceBasedSelectBoxEditor extends PureComponent {
 
         // Placeholder text must be unescaped in case html entities were used
         const placeholder = options && options.placeholder && i18nRegistry.translate(unescape(options.placeholder));
+        const loadingLabel = i18nRegistry.translate('loading', 'Loading', [], 'Neos.Neos', 'Main');
+        // const loadingLabel = 'Loading';
```
Comment (dimaip) [inline packages/neos-ui-editors/src/Editors/SelectBox/DataSourceBasedSelectBoxEditor.js]: somethins commented out

File: packages/react-ui-components/src/SelectBox/selectBox.js [hunk 4]
```diff
@@ -233,11 +239,12 @@ export default class SelectBox extends PureComponent {
             label = selectedValue.label;
             icon = selectedValue.icon ? selectedValue.icon : icon;
         } else if (displayLoadingIndicator) {
-            label = '[Loading]'; // TODO: localize
+            label = loadingLabel ? '[' + loadingLabel + ']' : '[Loading]';
         } else if (placeholder) {
             label = (<span className={theme.selectBox__placeholder}>{placeholder}</span>);
             icon = placeholderIcon ? placeholderIcon : icon;
         }
+        console.log('label', label);
```
Comment (dimaip) [inline packages/react-ui-components/src/SelectBox/selectBox.js]: console.log

**Pull request merged**